packages/dotnet-sdk/src/TeamsFx/Configuration/ConfigurationMethods.cs

```diff
@@ -31,7 +32,16 @@ public static class TeamsFxConfigurationMethods
         services.AddScoped<TeamsUserCredential>();
 
         services.AddOptions<AuthenticationOptions>().Bind(namedConfigurationSection.GetSection(AuthenticationOptions.Authentication)).ValidateDataAnnotations();
-
+        services.AddOptions<BotAuthenticationOptions>().Configure<IOptions<AuthenticationOptions>, IHttpContextAccessor> ((botAuthOption, authOptions, contextAccessor) => {
```

Comment:
Do we still need `IHttpContextAccessor`?

Reply:
No. Removed.

Reply:
It seems that there's still `IHttpContextAccessor`?

Reply:
Fixed.